### DISCUSSION TITLE
Anaconda package

### DISCUSSION THREAD

**Discussion by marickmanrho** [lacourjansenlab/NISE_2017#15] [Team Posts]
We could make NISE into a conda package to make it easy to install. Conda is most famous for Python packages, but it excels in packaging software with require compilation.  The requirements of NISE (FFTW, MPI, CMake, etc) are already available through Conda / Conda-forge. Especially now that the installation of NISE is done through CMake it shouldn't be too hard to do.

The package can then be uploaded to conda-forge which makes it a breeze to install on other machines from scratch.

A good starting point is [found here](https://medium.com/runic-software/simple-guide-to-conda-packages-4658cee798d6).

**Comment by GHlacour** [comment]
This sounds like a good idea! If anyone is interested in trying feel free to do so and keep us posted.
